modules/modules.properties

```diff
@@ -110,6 +110,7 @@ bundle.symbolic.name[fragment-web]=com.liferay.fragment.web
 bundle.symbolic.name[frontend-editor-alloyeditor-web]=com.liferay.frontend.editor.alloyeditor.web
 bundle.symbolic.name[frontend-editor-ckeditor-web]=com.liferay.frontend.editor.ckeditor.web
 bundle.symbolic.name[frontend-js-alert-support-web]=com.liferay.frontend.js.alert.support.web
+bundle.symbolic.name[frontend-js-api]=com.liferay.frontend.js
```

Comment:
another noob question here, what is this file for and why did you add this?

Reply:
I have no idea, but it lists all existing bundles. If you don't add the project here it's ignored by gradle.

Reply:
> i have no idea

![](https://media.giphy.com/media/Ae7SI3LoPYj8Q/giphy.gif)